### ISSUE TITLE
In v1, there were rendering options: wireframe, bounding box, helpers, focus object, debug layer. Where are they in v2?

### ISSUE THREAD

**Issue by julien-moreau** [BabylonJS/Editor#84]
In v1, there were rendering options: wireframe, bounding box, helpers, focus object, debug layer. Where are they in v2?

**Comment by julien-moreau** [comment]
Will add the toolbar :)

**Comment by Nodragem** [comment]
Maybe a menu in the preview window? I mean, the toolbar may not need to be always on-screen as it is not essential information?

**Comment by julien-moreau** [comment]
You mean like a widget ?

**Comment by Nodragem** [comment]
I am not familiar with the UI vocabulary :/ ... 
But here are examples from Blender, Unity and Godot:
![unity_2018-07-19_17-34-33](https://user-images.githubusercontent.com/10520249/42957319-3939c262-8b7a-11e8-8ea5-3fd8a47608c9.png)
![godot_v2 1 3-stable_win64_2018-07-19_17-33-08](https://user-images.githubusercontent.com/10520249/42957323-3be478d6-8b7a-11e8-87bb-c38f999ab8b4.png)
![blender_2018-07-19_17-35-24](https://user-images.githubusercontent.com/10520249/42957329-3f5a49aa-8b7a-11e8-9dc8-6ec24a8fce50.png)

Basically, that menu doesn't need to be on the screen at all time as in v1, the user can quickly show/hide it when they need it.

Concerning focus object, maybe it could be nice to have the option when you right-click an object in scenegraph too? maybe we could also focus by double-clicking an object in the scenegraph?

**Comment by julien-moreau** [comment]
Just added :)

**Comment by Nodragem** [comment]
Hello,
I just had a quick test; very nice integration!
Here is another little idea:
- would be nicer to have a little token (e.g. a check box or check-list symbol) next to each option that tells us which options are currently on/off

Now that there is a toolbar in the preview window, I also started a new issue on what could be put there.

All that are merely suggestions, :) so feel free to not listen to all of them.

**Comment by julien-moreau** [comment]
I 100% agree with the little idea and I tried to but my w2ui UI library didn’t want to work like in its demos. It is still in my todo list to find the problem
